Cura/util/meshLoaders/stl.py

```diff
@@ -22,17 +22,17 @@
 def _loadAscii(m, f):
 	cnt = 0
 	for lines in f:
-		for line in lines.split('\r'):
-			if 'vertex' in line:
+		for line in lines.split(b'\r'):
+			if b'vertex' in line:
 				cnt += 1
-	m._prepareFaceCount(int(cnt) / 3)
+	m._prepareFaceCount(int(int(cnt) / 3))
```

Comment:
I'm not so sure about that.

Reply:
You are right.
The type of int(cnt) / 3 is int in py2, float in py3 :(
Should explain and maybe fix #6.